build/dependencies.props

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
   <!-- Partner Versions-->
   <PropertyGroup>
-    <CodeAnalysisVersion>2.3.0-beta1</CodeAnalysisVersion>
+    <CodeAnalysisVersion>2.3.0-beta3-61731-06</CodeAnalysisVersion>
```

Comment:
I was going with the assumption that since Razor.Workspaces is using this build, it is the one we should use. But will confirm with the Roslyn folks.